lib/Runtime/Library/WebAssemblyMemory.h

```diff
@@ -9,6 +9,9 @@ namespace Js
 {
     class WebAssemblyMemory : public DynamicObject
     {
+    protected:
+        DEFINE_VTABLE_CTOR( WebAssemblyMemory, DynamicObject );
+        DEFINE_MARSHAL_OBJECT_TO_SCRIPT_CONTEXT( WebAssemblyMemory );
```

Comment:
Is this related to legacy no jit? I talked with @boingoing and I think there is more to do here.